felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -451,10 +453,12 @@ func newBPFEndpointManager(
 		dsrOptoutCidrs:          len(config.BPFDSROptoutCIDRs) > 0,
 		bpfExtToServiceConnmark: config.BPFExtToServiceConnmark,
 		psnatPorts:              config.BPFPSNATPorts,
-		bpfmaps:                 bpfmaps,
+		v4BPFMaps:               v4Maps,
+		v6BPFMaps:               v6Maps,
+		commonBPFMaps:           commonMaps,
```

Comment:
nit - feels like we do not have any other than bpf maps

```suggestion
		v4Maps:               v4Maps,
		v6Maps:               v6Maps,
		maps:                   commonMaps,
```

And I wonder if the v4/6 maps can't be just part of the `bpfEndpointManagerDataplane` and them you would access them like `m.v6.maps.ArpMap...`

In fact, if you embed the maps in the dataplane like 
```
type bpfEndpointManagerDataplane struct {
        *bpfmap.Maps                                                                <---
        
	ipFamily proto.IPVersion
	hostIP   net.IP
	mgr      *bpfEndpointManager

	ifaceToIpMap map[string]net.IP

	// IP of the tunnel / overlay device
	tunnelIP net.IP
}
```

You could just access them as `m.v6.ArpMap`, can't you?